tests/e2e/local/minikube/Troubleshooting.md

```diff
@@ -24,7 +18,7 @@ This file helps note down issues we have seen and how to debug them
    ```
 
 1. When running tests, if you get "Bad Request" error
-   Minikube uses insecure localregistry opened at localhost:5000, make sure you specify HUB as `localhost:5000` and TAG as `latest` when running the tests.
+   Minikube uses insecure local registry opened at localhost:5000, make sure you specify HUB as `localhost:5000` and TAG as `latest` when running the tests.
```

Comment:
This is out of date as well. Could you please remove or update?

Reply:
hmmmm, let me check the codes.

Reply:
@hklai 

After checking [minikube/setup_test.sh](https://github.com/istio/istio/blob/master/tests/e2e/local/minikube/setup_test.sh#L14-L17), I assume we still need this line?

Reply:
Check the line again. It is just make docker but it does not push to anywhere.

Docker just picks up the images locally. So the comment there is probably out of date too. =)

Reply:
Check the line again. It is just make docker but it does not push to anywhere.

Docker just picks up the images locally. So the comment there is probably out of date too. =)

Reply:
oh, just missing `make push`?

Thanks for reminding. I just fix that :) PTAL.